packages/node_modules/express-pouchdb/lib/index.js

```diff
@@ -185,6 +185,38 @@ module.exports = function (startPouchDB, opts) {
     next();
   });
 
+  app.use(function (req, res, next) {
+    var regexFindDbName = /\/([^\/]*)/
+    var dbName = decodeURIComponent(req.url.match(regexFindDbName)[1])
+
+    if (!dbName) {
+      return next();
+    }
+
+    // check for internal routes GET /_all_dbs
+    if (dbName[0] === '_') {
+      for (var i = 0, route; i < app._router.stack.length; i++) {
+        route = app._router.stack[i].route;
+        if (route) for (var j = 0, method; j < route.stack.length; j++) {
+          method = route.stack[j].method.toUpperCase()
+          if (method === req.method && route.path.substr(1, dbName.length) === dbName) {
+            console.log('ok?')
+            return next();
+          }
+        };
+      }
+    }
```

Comment:
There's a good reason that that particular filename sanitization is in there, though; you wouldn't want users to be able to create a DB called `../../etc/passwords` for instance. I think this needs a lot more thought and testing, and I'm also concerned about backwards compat. If we're going to break users who used to rely on `new PouchDB('foo/bar')` not actually creating a `foo/` directory then it should at least be a major version release.

In any case all of this code is very hairy and lacks tests, so maybe that's the first problem we should attempt to solve...